scripts/run-framework/run-framework.py

```diff
@@ -133,6 +133,9 @@ def setup_arg_parsing():
                         help='Logging level (default=%(default)s). '
                              'Options: debug, info, warning, error, critical')
 
+    parser.add_argument('--no_compress', dest='no_compress', action='store_true',
```

Comment:
I went with `no_compress` since I assumed we want to compress it by default since that was the preference when the compression was originally added. It can be easily reversed if you'd rather have it off by default